src/Workspaces/Core/Portable/Classification/ClassificationTypeNames.cs

```diff
@@ -27,6 +27,11 @@ public static class ClassificationTypeNames
         public const string StructName = "struct name";
         public const string TypeParameterName = "type parameter name";
 
+        public const string FieldName = "field name";
+        public const string MethodName = "method name";
+        public const string PropertyName = "property name";
+        public const string EventName = "event name";
```

Comment:
paging @DustinCampbell .  I think the primary concern we had here (and why we didn't do this earlier), is that we wanted flexibility to be able to classify things at a veyr fine grained level.  For example someone could want a set of classifications for "private static method" vs "public instance method".  I was prototyping being able to do this using the SymbolSpecification stuff @dpoeschl introduced for naming.  However, the primary blocker was that it would need a revamp of fonts/colors in VS to support this, even if we did have it at the data layer.

Reply:
@CyrusNajmabadi for context, we wanted to add these to the API so that VS 4 Mac doesn't lose the support they already had, even if VS doesn't make use of it right away.

Reply:
Classification types are a hierarchy, so this shouldn't block future work to have more detailed classifications.

Reply:
I think the concern was more that this was part of our public API.  So we wanted to get the public API shape right around classification.  But i agree that such a desire should not block pretty simple stuff that could be done now.  I just want Dustin involved so he's aware of what's going on and doesn't have any strong objections himself.

Reply:
Thanks @CyrusNajmabadi! Sorry for the delay (I've been OOF sick).

I'm generally OK with this. There are a lot of pivots we could consider here (read-only fields, different colors for different accessibility, etc., etc.), but I don't think that should block this. After all, we'd probably want to introduce more pivots for types as well and that's a similar concern.